biojava-structure/src/main/java/org/biojava/nbio/structure/SVDSuperimposer.java

```diff
@@ -131,11 +131,11 @@
  */
 public class SVDSuperimposer {
```

Comment:
Oh yes that's true. A possible clean solution to it would be to leave the interface of the current SVDSuperimposer untouched, whilst implementing a new SVDSuperimposer in geometry package that uses vecmath. Then the Atom-based SVDSuperimposer would simply wrap the Point3d based one.